package.json

```diff
@@ -50,6 +50,7 @@
     "filesize": "^10.1.0",
     "form-data": "^4.0.0",
     "isomorphic-dompurify": "^1.8.0",
+    "jszip": "^3.10.1",
```

Comment:
Changes to the package.json should have an accompanying change to the `yarn.lock`